Deartoday/Deartoday/Screen/Onboarding/Controller/OnboardingViewController.swift

```diff
@@ -7,10 +7,121 @@
+    @IBOutlet weak var labelBottomConstraint: NSLayoutConstraint!
+    
+    // MARK: - Life Cycle
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        setLayout()
+        setFirstLabelUI()
+        setSecondLabelUI()
+        setFirstAlpha()
+        setSecondAlpha()
+        setFirstAnimation()
+        imageGestrue()
+    }
+    
+    // MARK: - @objc
+    
+    @objc func circleButtonDidTap() {
+        print("로티")
+    }
+    
+    // MARK: - Custom Method
+    
+    @IBAction func nextButtonDidTap(_ sender: UIButton) {
+        setFirstAlpha()
+        setSecondAnimation()
+        showCircleButton()
+        isEnableBoxButton()
+    }
+    
+    @IBAction func boxButtonDidTap(_ sender: UIButton) {
+        print("로티")
+    }
+    
+    private func imageGestrue() {
+        let imageGesture = UITapGestureRecognizer(target: self, action: #selector(circleButtonDidTap))
+        circleButtonImage.addGestureRecognizer(imageGesture)
+        circleButtonImage.isUserInteractionEnabled = false
+    }
+    
+    private func setFirstLabelUI() {
+        firstLabelCollection.forEach {
+            $0.textColor = .white
+            $0.font = .p1
+            $0.font = .systemFont(ofSize: 13)
+            $0.textAlignment = .center
+        }
+    }
+    
+    private func setSecondLabelUI() {
+        secondLabel.textColor = .white
+        secondLabel.font = .p1
+        secondLabel.font = .systemFont(ofSize: 13)
+        secondLabel.textAlignment = .center
+    }
+    
+    private func setFirstAlpha() {
+        firstLabelCollection.forEach {
+            $0.alpha = 0
+        }
+        nextButton.alpha = 0
+    }
+    
+    private func setSecondAlpha() {
+        secondLabel.alpha = 0
+    }
+    
+    private func setFirstAnimation() {
+        UIView.animate(withDuration: 0.5, delay: 0.5, animations: {
+            self.firstLabelCollection[0].transform = CGAffineTransform(translationX: 0, y: -16)
+            self.firstLabelCollection[0].alpha = 1
+        }, completion: { _ in
+                       UIView.animate(withDuration: 0.5, delay: 0.5, animations: {
+            self.firstLabelCollection[1].transform = CGAffineTransform(translationX: 0, y: -16)
+            self.firstLabelCollection[1].alpha = 1
+                       }, completion: { _ in
+                           UIView.animate(withDuration: 0.3, delay: 0.3, animations: {
+                               self.nextButton.alpha = 1
+                           }, completion: { _ in })
+                       })
+    })
+    }
+    
+    private func setSecondAnimation() {
+        UIView.animate(withDuration: 0.5, animations: {
+            self.secondLabel.transform = CGAffineTransform(translationX: 0, y: -16)
+            self.secondLabel.alpha = 1
+        }, completion: { _ in })
+    }
+    
+    private func setLayout() {
+        let isSe2 = getDeviceHeight() == 667
+        labelBottomConstraint.constant = isSe2 ? 45 : 69
```

Comment:
이거도 아까 제가 이렇게 써드리긴 했는데
```suggestion
        labelBottomConstraint.constant = (getDeviceHeight() == 667) ? 45 : 69
```
이렇게도 가능하겠네융
가독성도 중요하고.. 그렇긴 한데 (혹은 나중에 extension함수로 se2 핸드폰인지 알아보는 함수 넣어둘 수 있음)
한번만 쓰이는 변수는 안쓸 수도 있으니까!